README.md

```diff
@@ -54,6 +59,30 @@ corresponding `has_` method instead (e.g., `has_position()`):
     if not snapshot.has_mass():
         snapshot.mass = [2.,2.,10.]
 
+## Topology
+
+The particle topology is stored in `Topology` derived classes. `Topology` derived
+classes are `Bonds`, `Angles`, `Dihedrals`, and `Impropers`.
+
+These constructor arguments are available to each type of topology as attributes:
+
+- `N`: number of topology, e.g., bonds, angles, dihedrals, or impropers (int)
+- `num_types`: number of topology types (int)
```

Comment:
A comment that type deducation works the same for these objects as for the Snapshot would also be good.